resources/views/ta/modules/sections.blade.php

```diff
@@ -352,8 +355,42 @@
         </div>
     </div>
 </div>
+
 </div>
 
+<div id="bulkCheckinModal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                <h4 class="modal-title">Bulk Checkin</h4>
+            </div>
+            <form class="form" method="POST" action="{{ route("tasectionbulkcheckin") }}">
+
+                <input id="bulkCheckinInputHidden" type="hidden" name="_token" value="{{ csrf_token() }}" />
+                <input type="hidden" id="bulkCheckinInputSID" name="inputSID" value="" />
+
+                <div id="pickDateContainer"></div>
+                <div class="modal-body">
+                    <p>This checks in all academic interns assigned to the section. </p>
+                    <div class="form-group">
+                        <label>Date: </label>
+                        <input type="text" class="form-control" name="inputDate" id="inputDate" placeholder="Date" readonly />
+                    </div>
+                </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                <input type="submit" class="btn btn-success" value="Bulk Checkin" />
+            </div>
+            </form>
+        </div>
+    </div>
+</div> 
+
+<script src="/packages/pickadate/picker.js"></script>
```

Comment:
These are included in the base template located here:

https://github.com/Cal-CS-61A-Staff/Check-In/blob/4cd90947364ecf480af61010873b62ca8c3913db/resources/views/core/footer.blade.php#L14